doc/plugins.md

```diff
@@ -166,6 +166,8 @@ The list of plugins:
     — sanitize HTML
 *   [`rehype-section`](https://github.com/agentofuser/rehype-section)
     — wrap headings and their contents into nested `<section>` elements
+*   [`rehype-sectionize`](https://github.com/hbsnow/rehype-sectionize)
+    — when heading has id, section with id wraps the heading and content
```

Comment:
I don’t understand this text. Perhaps something like the following, based on the description of `rehype-section`.

```suggestion
    — wrap headings (with IDs) and their contents into nested `<section>`
    elements, with attributes
```

But like, what does your plugin do when there are no IDs on headings? Maybe you don’t need to include the `(with IDs)` part here.

Reply:
I'm not very good at English, thank you!